Standards/scs-XXXX-vN-Availability-Zones-Standard.md

```diff
@@ -0,0 +1,196 @@
+The result should enable CSPs to know when to create AZs to be SCS-compliant.
+
+## Design Considerations
+
+Availability Zones should represent parts of the same physical deployment that are independent of each other.
+The maximum of physical independence is achieved through putting physical machines into different fire zones.
+In that case a failure case up to level 3 as described in the taxonomy of failure safety levels document[^1] will not lead to a complete outage of the deployment.
+
+Having Availability Zones represent fire zones will also result in AZs being able to take workload from another AZ in a Failure Case of Level 3.
+So that even the destruction of one Availability Zone will not automatically include the destruction of the other AZs.
+
+:::caution
+
+Even with fire zones being physically designed to protect parts of a data center from severe destruction in case of a fire, this will not always succeed.
+Availability Zones in Clouds are most of the time within the same physical data center.
+In case of a big catastrophe like a huge fire or a flood the whole data center could be destroyed.
+Availability Zones will not protect customers against these failure cases of level 4 of the taxonomy of failure safety[^1].
+
+:::
+
+Smaller deplyoments like edge deployments may not have more than one fire zone in a single location.
+To include such deployments, it should not be required to use Availability Zones.
+
+Other physical factors that should be considered are the power supplies, internet connection, cooling and core routing.
+Availability Zones have been also being configured to show redundancy in e.g. Power Supply as in the PDU.
+That means there are deployments, which have Availability Zones per rack as each rack has it's own PDU and this was considered to be the single point of failure an AZ should represent.
+While this is also a possible measurement of independency it only provides failure safty for level 2.
+Therefore this standard should be very clear about which independency an AZ should represent and it should not be allowed to have different deployments with their Availability Zones representing different levels of failure safety.
+
+Additionally Availability Zones are available for Compute, Storage and Network services.
+They behave differently for each of these resources and also when working across resource-based Availability Zones, e.g. attaching a volume from one AZ to a virtual machine in another AZ.
+For each of these IaaS resource classes, it should be defined, under which circumstances Availablitiy Zones should be used.
+
+### Scope of tha Availability Zone Standard
+
+When elaborating redundancy and failure safety in data centers, it is necessary to also define redundancy on the physical level.
+There are already recommendations from the BSI for physical redundancy within a cloud deployment [^2].
```

Comment:
Add BSI to the Terminology glossary? It is only mentioned but never explained.

Reply:
I added it.